js/packages/mobile-wallet-adapter-walletlib/android/src/main/java/com/solanamobile/mobilewalletadapterwalletlib/reactnative/SolanaMobileWalletAdapterWalletLibModule.kt

```diff
@@ -248,6 +245,59 @@ class SolanaMobileWalletAdapterWalletLibModule(val reactContext: ReactApplicatio
         }
     }
 
+    /* Client Verification Utils */
```

Comment:
the methods here are exposing both the android PackageManager class and our Digital Asset Links library. It might make sense to fork this out as its own react native library (likely a very small lib) tho its probably not worth the effort at this time. 

Proposal: move this to its own native lib, and expose it in its won js file (ie useClientTrust.ts). this will make it easier to fork out later